sentry-kotlin-multiplatform/src/commonTest/kotlin/io/sentry/kotlin/multiplatform/SentryE2ETest.kt

```diff
@@ -0,0 +1,130 @@
+package io.sentry.kotlin.multiplatform
+
+import io.ktor.client.HttpClient
+import io.ktor.client.request.get
+import io.ktor.client.request.headers
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.HttpHeaders
+import io.sentry.kotlin.multiplatform.utils.realDsn
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNotNull
+import kotlin.time.Duration.Companion.seconds
+
+@Serializable
+private data class SentryEventSerializable(
+    val id: String? = null,
+    val project: Long? = null,
+    val release: String? = null,
+    val platform: String? = null,
+    val message: String? = "",
+    val tags: List<TagSerializable> = listOf(),
+    val fingerprint: List<String> = listOf(),
+    val level: String? = null,
+    val logger: String? = null,
+    val title: String? = null
+)
+
+@Serializable
+private data class TagSerializable(
+    val key: String? = null,
+    val value: String? = null
+)
+
+class SentryE2ETest : BaseSentryTest() {
+    private val client = HttpClient()
+    private val jsonDecoder = Json { ignoreUnknownKeys = true }
+    private var sentEvent: SentryEvent? = null
+
+    @BeforeTest
+    fun setup() {
+        assertNotNull(authToken)
+        sentryInit { options ->
+            options.dsn = realDsn
+            options.beforeSend = { event ->
+                sentEvent = event
+                event
+            }
+        }
+    }
+
+    private suspend fun fetchEvent(eventId: String): String {
+        val url =
+            "https://sentry.io/api/0/projects/sentry-sdks/sentry-kotlin-multiplatform/events/$eventId/"
```

Comment:
M: Lets do similar to Dart, parametrize the URL.
https://github.com/getsentry/sentry-dart/blob/99490581cac83f47aff9c6d3186d401d3dbc55db/e2e_test/bin/e2e_test.dart#L99
Put the org and project closer to the DSN, so in case we change it, we know that we have to change the project and org too instead of only DSN.